templates/repo/branch_dropdown.tmpl

```diff
@@ -70,8 +70,8 @@
 <div class="js-branch-tag-selector {{if .ContainerClasses}}{{.ContainerClasses}}{{end}}">
 	{{/* show dummy elements before Vue componment is mounted, this code must match the code in BranchTagSelector.vue */}}
 	<div class="ui dropdown custom">
-		<button class="branch-dropdown-button gt-ellipsis ui basic small compact button gt-df gt-m-0">
-			<span class="text gt-df gt-ac gt-mr-2">
+		<button class="branch-dropdown-button ui basic small compact button gt-df gt-m-0">
+			<span class="text gt-df gt-ac gt-mr-2" style="overflow: hidden">
```

Comment:
We could add a `gt-overflow-hidden` to `helpers.css`, better than inline style at least.

Reply:
I found an existing utility class that worked. I see there's already been discussion about using Tailwind too.